Code/GraphMol/FMCS/FMCS.cpp

```diff
@@ -45,7 +53,7 @@ void MCSParameters::setMCSAtomTyperFromEnum(AtomComparator atomComp) {
 
 void MCSParameters::setMCSAtomTyperFromConstChar(const char* atomComp) {
   PRECONDITION(atomComp, "atomComp must not be NULL");
-  static const std::map<const char*, AtomComparator> atomCompStringToEnum = {
+  static const std::map<const char*, AtomComparator, cmpCStr> atomCompStringToEnum = {
       {"Any", AtomCompareAny},
```

Comment:
What does this change have to do with the optional compilation?

Reply:
This does not have to do with the optional compilation _per se_, but with exposing MCS functionality to JS. `MCSParameters` are parsed from a JSON string and I found out that the JSON parser was broken for `AtomCompare` and `BondCompare`, so I fixed it and added unit tests.